# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../checkpassword/head_n.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.orig	2005-02-23 14:39:41.260995576 -0300
+++ ./Makefile	2005-02-23 14:40:11.397414144 -0300
@@ -22,7 +22,7 @@
 
 auto_home.c: \
 auto-str conf-home
-	./auto-str auto_home `head -1 conf-home` > auto_home.c
+	./auto-str auto_home `head -n 1 conf-home` > auto_home.c
 
 auto_home.o: \
 compile auto_home.c
@@ -91,7 +91,7 @@
 choose: \
 warn-auto.sh choose.sh conf-home
 	cat warn-auto.sh choose.sh \
-	| sed s}HOME}"`head -1 conf-home`"}g \
+	| sed s}HOME}"`head -n 1 conf-home`"}g \
 	> choose
 	chmod 755 choose
 
@@ -166,7 +166,7 @@
 warn-auto.sh conf-ld
 	( cat warn-auto.sh; \
 	echo 'main="$$1"; shift'; \
-	echo exec "`head -1 conf-ld`" \
+	echo exec "`head -n 1 conf-ld`" \
 	'-o "$$main" "$$main".o $${1+"$$@"}' \
 	) > load
 	chmod 755 load
--- ./print-cc.sh.orig	2005-02-23 14:39:48.629875336 -0300
+++ ./print-cc.sh	2005-02-23 14:40:16.423650040 -0300
@@ -1,4 +1,4 @@
-cc="`head -1 conf-cc`"
+cc="`head -n 1 conf-cc`"
 systype="`cat systype`"
 
 cat warn-auto.sh
